README.md

```diff
@@ -12,6 +12,7 @@ Use GitHub repositories to document everything. \o/
 - https://github.com/joshwnj/knowledge
 - https://github.com/joshpierro/knowledge
 - https://github.com/nikitavoloboev/knowledge-map
+- https://github.com/galileo/knowledge
```

Comment:
Ohh thats the secret algorithm 😄  I will change this! 👍 
Thank you @RichardLitt